### PR TITLE
{data}[intel/2018b] PyTables v3.5.2 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegel how many versions of PyTables do you really need? I reviewed at least a hundred easyconfigs for it already :wink:

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node118.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/c580d6f684aa4ed9ccb278bc6f76446f for a full test report.

File: easybuild/easyconfigs/p/PyTables/PyTables-3.5.2_fix-LIBS.patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
+PyTables expects that LIBS is a list of library names, and will add '-l' by itself.
```
Comment (lexming) [inline easybuild/easyconfigs/p/PyTables/PyTables-3.5.2_fix-LIBS.patch:1]: This patch is identical to `PyTables-3.5.2-fix-libs.patch`, which is already in the tree. Please switch to it and remove the duplicate from your PR.

**Comment by boegel** [comment]
Old software versions + deprecated toolchain version, so closing this

**Pull request closed without merging**